src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
```

Comment:
The BEM methodology suggests that the block should be named according to its purpose. The class 'stopwatch' is good, but ensure that it represents the main purpose of the block and is not too generic.